### DISCUSSION TITLE
Thanks for keeping the App alive

### DISCUSSION THREAD

**Discussion by hofq** [freelensapp/freelens#57] [General]
Hey @dex4er

I've just wanted to say thank you for keeping the App alive. Tried all the Tools on the Market (including the Closed Source Lens) but none of them have kept up to the workflow speed i had with this tool.

I hope i can support in the future

**Comment by robertobandini** [comment]
> Hey @dex4er
> 
> I've just wanted to say thank you for keeping the App alive. Tried all the Tools on the Market (including the Closed Source Lens) but none of them have kept up to the workflow speed i had with this tool.
> 
> I hope i can support in the future

Hi, as the creator of this initiative I thank you in turn for your intervention and your interest. If you want to contribute let us know or propose your issues directly, we are looking for collaborators to carry on the project each with a little of their time. Thank you!
